.pre-commit-config.yaml

```diff
@@ -1,28 +1,35 @@
-exclude: ^secrets/|^appenv$|pkgs/fc/sensusyntax/fixtures/(syntaxerror|empty).json|(tests/testlib.nix|nixos/roles/devhost/vm.nix|tests/physical-installer.nix)|^changelog.d/new_fragment.md.j2$
+exclude: ^secrets/|^appenv$|pkgs/fc/sensusyntax/fixtures/(syntaxerror|empty).json|(tests/testlib.nix|nixos/roles/devhost/vm.nix|tests/physical-installer.nix)
 repos:
 - hooks:
+  - id: detect-private-key
+  - id: check-added-large-files
   - exclude: "(?x)^(\n  secrets/|environments/.*/secret.*|\n  .*\\.patch\n)$\n"
     id: trailing-whitespace
   - exclude: "(?x)^(\n  environments/.*/secret.*|\n  .*\\.patch\n)$\n"
     id: end-of-file-fixer
-  - id: check-yaml
-  - id: check-added-large-files
-  - id: check-json
-  - id: check-xml
-  - id: check-toml
-  - id: detect-private-key
+  - exclude: "(?x)^(\n  environments/.*/secret.*|\n  .*\\.patch\n)$\n"
+    id: check-yaml
+  - exclude: "(?x)^(\n  environments/.*/secret.*|\n  .*\\.patch\n)$\n"
+    id: check-json
+  - exclude: "(?x)^(\n  environments/.*/secret.*|\n  .*\\.patch\n)$\n"
+    id: check-xml
+  - exclude: "(?x)^(\n  environments/.*/secret.*|\n  .*\\.patch\n)$\n"
+    id: check-toml
   repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v5.0.0
 - hooks:
-  - args: ["--profile", "black", "--filter-files"]
+  - args:
+    - --profile
+    - black
+    - --filter-files
     id: isort
     name: isort (python)
   repo: https://github.com/pycqa/isort
-  rev: 6.0.1
+  rev: 5.13.2
```

Comment:
@ctheune IIRC you just updated that recently, so maybe we'd like to keep the update and bump versions in the code-style-template instead?

Reply:
Done in https://github.com/flyingcircusio/codestyle-template/pull/11